ATM.py

```diff
@@ -0,0 +1,106 @@
+"""
+
+The script simulates ATM that preforms the following:
+balance check, withdraw cash, deposit cash and
+password change.
+
+"""
+import os
+import sys
+
+
+PATH=1
+BALANCE=1
+PASSWORD=0
+
+
+def file_to_dic(filename):
+    """
+
+    :arg
+        filename(.txt file) - parameter
+
+    :return:
+        a dictionary, key=id and value=[password, balance]
+
+    """
+    atm = {}
+    with open(filename, 'r') as file:
+        for line in file:
+            details = line.strip().split(' ')
+            atm[details[0]] = details[1:]
```

Comment:
add in the documentation an explanation of the content format that you work with